File: Javascript/Fader_In.js
﻿#pragma strict

var loadGUI : GUITexture;

function Start () {
	var currentRes : Rect = Rect (-Screen.width * 0.5, -Screen.height * 0.5, Screen.width, Screen.height);
	
	guiTexture.pixelInset = currentRes;
}

function Update () 
{

}

function letsBegin()
	{
		Destroy(gameObject);
		Application.LoadLevel("Island");
		
	}

File: C#/Fader_In.cs
﻿using UnityEngine;
using System.Collections;

public class Fader_In : MonoBehaviour {
	
	public GUITexture fadeWhite;

	// Use this for initialization
	void Start () {
		Rect currentRes = new Rect (-Screen.width * 0.5f, -Screen.height * 0.5f, Screen.width, Screen.height);
		guiTexture.pixelInset = currentRes;
	}
	
	// Update is called once per frame
	void Update () 
	{
	
	}
	
	void LetsBegin()
	{
		//Instantiate(fadeWhite);
		Application.LoadLevel("Island");
		
	}
}
